packages/@react-spectrum/picker/docs/Picker.mdx

```diff
@@ -168,6 +169,23 @@ Picker supports the `name` prop for integration with HTML forms. The `key` of th
 </Picker>
 ```
 
+## Links
+
+By default, interacting with an item in a Picker triggers `onSelectionChange`. Alternatively, items may be links to another page or website. This can be achieved by passing the `href` prop to the `<Item>` component. Link items in a `Picker` are not selectable.
+
+```tsx example
+<Picker label="Project">
+  <Item href="https://example.com/" target="_blank">Create new…</Item>
```

Comment:
Do we want to encourage this pattern?

Reply:
I think probably not... though cat may already be out of the bag in regards to this pattern?